playwright/navigation.spec.ts

```diff
@@ -0,0 +1,17 @@
+import { expect, test } from '@playwright/test';
+
+test('Navigate and check content', async ({ page }) => {
+  await page.goto('/');
+  await expect(page.getByTestId('myHero')).toBeVisible();
+  await expect(page.getByTestId('whoAmI')).toBeVisible();
+  await expect(page.getByTestId('myWork')).toBeVisible();
+  await expect(page.getByTestId('footer')).toBeVisible();
+
+  await page.getByRole('link', { name: 'My work' }).click();
+  await page.waitForURL('/#portfolio');
+  await expect(page.getByTestId('myWork')).toBeInViewport();
```

Comment:
these are 2 different strings here, probably could be better

Reply:
Changed the `link` and the `data-test-id` to "portfolio"
commit [e228460](https://github.com/upleveled/portfolio-cms/pull/112/commits/e22846068bc93a4f8055076a6634814b277d3b1b)